recipe/meta.yaml

```diff
@@ -22,27 +22,32 @@ requirements:
     run:
         - python
         - numpy x.x
-        - scipy
+        - scipy  # 0.16.0
         - pillow
         - matplotlib
-        - scikit-learn
-        - scikit-image
         - pyproj
-        - joblib
+        - scikit-image
         - simplekml
+        - joblib  0.7.1
```

Comment:
@dbuscombe-usgs same question as above as `joblib` is in its `0.10.0`. But here it is not a problem as nothing is downgraded and `joblib 0.7.1` is available in `conda-forge`.